### ISSUE TITLE
Build issue Makefile:3822: Makefile.deps: No such file or directory

### ISSUE THREAD

**Issue by haampie** [besser82/libxcrypt#173]
Looks like something has regressed in libxcrypt 4.4.35 + perl 5.38 + gmake 4.3.

```
Makefile.deps
```

is not created, or there's a race?

---

Actually there's already an issue with configure when I don't disable dependency tracking:

```
checking that generated files are newer than configure... done
configure: creating ./config.status
config.status: creating Makefile
config.status: creating libxcrypt.pc
config.status: creating config.h
config.status: config.h is unchanged
config.status: executing depfiles commands
config.status: error: in `/tmp/harmen/spack-stage/spack-stage-libxcrypt-4.4.35-4bytcldeuh7kp3l4hu6wl6yqaooq2qs4/spack-src':
config.status: error: Something went wrong bootstrapping makefile fragments
    for automatic dependency tracking.  If GNU make was not used, consider
    re-running the configure script with MAKE="gmake" (or whatever is
    necessary).  You can also try re-running configure with the
    '--disable-dependency-tracking' option to at least be able to build
    the package (albeit without support for automatic dependency tracking).
See `config.log' for more details
```

**Comment by haampie** [comment]
Ah, I didn't realize https://github.com/besser82/libxcrypt/commit/95d6e03ae37f4ec948474d111105bbdd2938aba2 was not part of a tagged release.

**Comment by thesamesam** [comment]
@besser82 Would you be able to cut a new release for it? Thanks!

**Comment by besser82** [comment]
Fixed in [libxcrypt v4.4.36](https://github.com/besser82/libxcrypt/releases/tag/v4.4.36).
